pkg/rootless/rootless_linux.go

```diff
@@ -166,7 +166,8 @@ func GetConfiguredMappings() ([]idtools.IDMap, []idtools.IDMap, error) {
 	}
 	mappings, err := idtools.NewIDMappings(username, username)
 	if err != nil {
-		logrus.Errorf("cannot find mappings for user %s: %v", username, err)
+		logrus.Errorf(
+			"cannot find mappings for user %s: %v, examine rootless mode in man podman", username, err)
```

Comment:
This is used by more than just Podman, so this should probably be reworded - maybe `cannot find UID/GID mappings for user %s: %v - check /etc/subuid and /etc/subgid`?

Reply:
@mheon how about:
"cannot find UID/GID for user %s: %v - check rootless mode in the official documentation"

Reply:
I would be more specific.  `check rootless mode in man pages.`